patcher/src/ts/content/chat/ParseEmoji.tsx

```diff
@@ -59,7 +139,7 @@ function fromText(text: string, keygen: () => number) : JSX.Element[] {
 }
 
 function createRegExp() : RegExp {
-  return /[3]*[;:8][-']*[()@oO#$*pPD/|><]|\([6aAhH]\)|-_-zzZ|\(-_-\)zzZ|[oO]\.[oO]|:[a-z]*:/g;
+  return /:[a-zA-Z0-9]+:|[3]*[;:8][-']*[()@oO#$*pPD/|><]|\([6aAhH]\)|-_-zzZ|\(-_-\)zzZ|[oO]\.[oO]/g;
```

Comment:
changed regex to mehuge/agoknee recommendation to allow :pvp: to not return :P + "vp:"